src/locales/en.json

```diff
@@ -35,7 +35,7 @@
     "contributeProject": "miss your project?",
     "contributeCompany": "miss your company?",
     "tip": "miss your item?",
-    "introduction": "Welcome to OpenInsight-Index, where Chinese companys, projects and robot accounts are ranked by the activity and influence on GitHub, the leading source code repository for free and open source (FOSS) projects.",
+    "introduction": "Welcome to OpenInsight-Index, where Chinese companys, projects and robot accounts are ranked by the activity and influence on GitHub.",
```

Comment:
After a quick search, I found many keys in the two locale files are no longer used. Maybe we can delete the unused ones later.